core/templates/components/shared-component.module.ts

```diff
@@ -136,7 +136,8 @@ const firebaseAuthProviders = AuthService.firebaseAuthIsEnabled ? [
     StorySummaryTileDirective,
     SubtopicSummaryTileDirective,
     TakeBreakModalComponent,
-    TranslatePipe
+    TranslatePipe,
+    SummaryListHeaderComponent
```

Comment:
Can you move this to Line 138, in between `SubtopicSummaryTileDirective` and `TakeBreakModalComponent`

Reply:
Done.